CMakeLists.txt

```diff
@@ -80,11 +109,6 @@ include(GetVersionFromFile)
 message(STATUS "Project name: ${PROJECT_NAME}")
 message(STATUS "Project version: ${PROJECT_VERSION_STRING_FULL}")
 
-if(NOT COMPILE_WASM)
-  # Set BUILD_ARCH to native only while compiling for non wasm platform
-  set(BUILD_ARCH native CACHE STRING "Compile for this CPU architecture.")
-endif()
```

Comment:
https://github.com/browsermt/bergamot-translator/runs/4523341672?check_suite_focus=true#step:9:78

Any idea what's triggering `native` is coming in despite the above lines being removed?

Reply:
The lines above are moved to the top of the `CMakeLists.txt`. Essentially what this patch does, is keep old behaviour unchanged and try to get Windows compilation to accept some clang/g++ like `-march` flags.

Reply:
Oops. My bad. Resolving.